examples/gcd/sky130_floorplan.py

```diff
@@ -73,6 +71,9 @@ def place_pdn(fp, vdd, vss, hwidth, hspacing, hlayer, vwidth, vspacing,
    fp.place_wires([vss] * ngnd, core_left, bottom, 0, spacing,
                     core_w, stripe_w, stripe_layer, 'followpin')
 
+
+   # move here until test can be rebuilt
+   from siliconcompiler.floorplan import _layer_i
```

Comment:
Should we maybe delete these examples, or are we planning to try re-implementing the Floorplan API in the near future? They'll still be in the git history if we remove them. Ditto for the other tests.

Reply:
@WRansohoff they will be re-implemented in the near future, so it seemed easier to keep them around as a starting point.